hts-precompile/IHederaTokenService.sol

```diff
@@ -392,4 +392,55 @@ interface IHederaTokenService {
     /// @param serialNumber The serial number of the NFT to transfer.
     function transferNFT(address token,  address sender, address recipient, int64 serialNumber) external
         returns (int responseCode);
+
+    /// Allows spender to withdraw from your account multiple times, up to the value amount. If this function is called 
+    /// again it overwrites the current allowance with value.
+    /// Only Applicable to Fungible Tokens
+    /// @param token The hedera token address to approve
+    /// @param spender the account authorized to spend
+    /// @param amount the amount of tokens authorized to spend.
```

Comment:
nit: can we add the unit for this and other amounts. Devs may not know if its a gas unit or an hbar unit

Reply:
Ignore. Mixed items up